internal/service/anycast/api_anycast_config_resource_test.go

```diff
@@ -18,16 +18,18 @@ import (
 func TestAccOnPremAnycastManagerResource_basic(t *testing.T) {
```

Comment:
All tests in this file must be renamed

```suggestion
func TestAccAnycastConfigResource_basic(t *testing.T) {
```

Reply:
done